src/sentry/models/release.py

```diff
@@ -388,7 +388,7 @@ def set_commits(self, commit_list):
         """
 
         # Sort commit list in reverse order
-        commit_list.sort(key=lambda commit: commit.get("timestamp"), reverse=True)
+        commit_list.sort(key=lambda commit: commit.get("timestamp", 0), reverse=True)
```

Comment:
`0` is a sane default here since it will be lower than any other timestamp.

Also I think this is probably only happening in tests.